views/event_details.hbs

```diff
@@ -1,25 +1,21 @@
-<div class="event-details">
-  <div class="eventDetailsHeader">
-    <div>
-      {{>header}}
-    </div>
-    <div class="prevBtn ">
-      <a href="{{{back}}}"><span class="glyphicon glyphicon-chevron-left"></span></a>
-    </div>
-  </div>
-  <div class="event-details text-center">
-    <h1 class="text-warning eventName">{{event.en.name}}</h1>
-    {{#if imageUrl}}
-    <img src="{{event.imageUrl}}" alt="event picture">
-    {{/if}}
-    <h4 class="text-muted eventStartDate">{{prettyDate event.startTime}}</h4>
-    <h5 class="text-muted eventTime">{{prettyTime event.startTime}} - {{prettyTime event.endTime}}</h5> <h3 class="text-primary">at: {{event.placeId.en.name}}</h3>
+<div>
+  {{>header}}
+</div>
+<div class="prevBtn ">
+  <a href="{{{back}}}"><span class="glyphicon glyphicon-chevron-left"></span></a>
+</div>
```

Comment:
it could be something like
```html
<div class="eventDetailsHeader">

    {{>header}}
    <div class="prevBtn ">
     <a href="{{{back}}}"><span class="glyphicon glyphicon-chevron-left"></span></a>
    </div>
 </div>
```